app/plugins.js

```diff
@@ -307,6 +311,13 @@ function requirePlugins() {
 
       // populate the name for internal errors here
       mod._name = basename(path);
+      try {
+        // eslint-disable-next-line import/no-dynamic-require
+        mod._version = require(resolve(path, 'package.json')).version;
```

Comment:
maybe not an issue since it's (maybe?) cached, and looking at your screenshot it doesn't even take 10ms